### ISSUE TITLE
text to speech should read timestamps less literally

### ISSUE THREAD

**Issue by 573** [gabeg805/NFC-Alarm-Clock#15]
If configurable, tts should not read any punctuation potentially existing in the timestamp it gets.

Example for the time 10h19, currently tts reads ordinal digits (german locale): "zehnt uhr neunzehnt" (while technically not wrong, this is not used in spoken language) would roughly translate to "tenth nineteenth" and might be caused by punctuation following the digits in the timestamps.

I guess tts just reads what it gets, not parsing ordinal whatever, so a filter might be needed before feeding into tts.

**Comment by gabeg805** [comment]
Hi @573, yeah the way I had it reading things was like this:

The time, is, 10, 19.

The excessive commas are because I found it read pretty fast without the commas, so I needed to slow it down a bit.  It's kind of hacky and there's probably a better way to slow it down.

I just added a German translation, but full disclosure, I don't speak German.  I had to use Google Translate so I don't know how good it's going to be, but I'm always open to changing things and making it better!  With respect to TTS though, I did try to update it per your suggestion.  What it's reading now is:

Es ist 10:19.

Hopefully because there's not crazy punctuation and it's obviously a time, TTS will read it better.
